src/puter-js/src/modules/AI.js

```diff
@@ -123,56 +123,76 @@ class AI{
     }
 
     txt2speech = async (...args) => {
-        let MAX_INPUT_SIZE = 3000;
         let options = {};
         let testMode = false;
 
         if(!args){
             throw({message: 'Arguments are required', code: 'arguments_required'});
         }
+        
+        // Accept arguments in the following formats:
+        // 1. Verbose API
+        //      puter.ai.txt2speech("Hello world", {
+        //           voice: "Joanna",
+        //           engine: "neural",
+        //           language: "en-US"
+        //      })
+        // 2. Shorthand API
+        //      puter.ai.txt2speech("Hello world")
+        // 
+        // Default values:
+        //      voice: "Joanna"
+        //      engine: "standard"
+        //      language: "en-US"
+
 
-        // if argument is string transform it to the object that the API expects
         if (typeof args[0] === 'string') {
             options = { text: args[0] };
         }
 
-        // * ai.txt2speech('Hello, world!', 'en-US')
-        // * ai.txt2speech('Hello, world!', 'en-US', 'Brian')
-        if (args[1] && typeof args[1] === 'string') {
-            
-            // Determine language
-            if (args[1] && typeof args[1] === 'string') {
-                // Check if it's a language code (ISO 639-1 or with region)
-                // Pattern matches: en, es, fr, de, en-US, es-ES, fr-FR, etc.
-                const languageCodePattern = /^[a-z]{2}(-[A-Z]{2})?$/;
-
-                // if language code is invalid, throw an error
-                if(!languageCodePattern.test(args[1])){
-                    throw { message: 'Invalid language code', code: 'invalid_language_code' };
-                }
+        // Check if second argument is an options object
+        if (args[1] && typeof args[1] === 'object' && !Array.isArray(args[1])) {
+            Object.assign(options, args[1]);
+        } else if (args[1]) {
+            // If second argument is not an object, throw an error
+            throw { message: 'Second argument must be an options object. Use: txt2speech("text", { voice: "name", engine: "type", language: "code" })', code: 'invalid_arguments' };
```

Comment:
When changing methods on `puter.js`, we absolutely **must** be backwards-compatible. Apps on Puter will get these updates automatically, so regressions break apps.
![image](https://github.com/user-attachments/assets/136095b6-d5da-4184-8aa5-a75588b9a89e)

We can still have this options object (and I think it's a good idea to keep it), but the old format must work too.